src/read.jl

```diff
@@ -475,6 +500,31 @@ function reformat_demand_flexibility_input(
             # store to new df object
             demand_flexibility_updated[field] = eq_bus_df
 
+            # if flex amount specified in DOE data, re-format using bus EIA ID
+        elseif demand_flexibility.source == "doe" &&
+                field in ["flex_amt_up", "flex_amt_dn"]
+            
+            # all load buses with valid EIA ID 
+            flexible_bus_idx = intersect(sets.load_bus_idx, findall(case.bus_eiaid .> 0))
+            flexible_bus_num = length(flexible_bus_idx)
+            
+            # bus flexibility Percentage matrix
+            doe_bus_flexibility = zeros(size(case.demand, 1), flexible_bus_num)
+            doe_eiaid = [parse(Int, x) for x in names(fh)[2:end]]
```

Comment:
I know this was introduced before this PR, but can we change the name of the `fh` variable to something a little more meaningful?

Reply:
sure